libs/java/client_common/src/main/java/com/yahoo/athenz/common/utils/SSLUtils.java

```diff
@@ -0,0 +1,280 @@
+        public ClientSSLContextBuilder keyStorePassword(final char[] keyStorePassword) {
+            this.keyStorePassword = keyStorePassword;
+            return this;
+        }
+        
+        public ClientSSLContextBuilder keyManagerPassword(final char[] keyManagerPassword) {
+            this.keyManagerPassword = keyManagerPassword;
+            return this;
+        }
+        
+        public ClientSSLContextBuilder keyStorePath(final String keyStorePath) {
+            this.keyStorePath = keyStorePath;
+            return this;
+        }
+        
+        public ClientSSLContextBuilder keyStoreType(final String keyStoreType) {
+            this.keyStoreType = keyStoreType;
+            return this;
+        }
+        
+        public ClientSSLContextBuilder trustStorePath(final String trustStorePath) {
+            this.trustStorePath = trustStorePath;
+            return this;
+        }
+        
+        public ClientSSLContextBuilder trustStorePassword(final char[] trustStorePassword) {
+            this.trustStorePassword = trustStorePassword;
+            return this;
+        }
+        
+        public ClientSSLContextBuilder trustStoreType(final String trustStoreType) {
+            this.trustStoreType = trustStoreType;
+            return this;
+        }
+        
+        public ClientSSLContextBuilder keyStorePasswordAppName(final String keyStorePasswordAppName) {
+            this.keyStorePasswordAppName = keyStorePasswordAppName;
+            return this;
+        }
+        
+        public ClientSSLContextBuilder keyManagerPasswordAppName(final String keyManagerPasswordAppName) {
+            this.keyManagerPasswordAppName = keyManagerPasswordAppName;
+            return this;
+        }
+        
+        public ClientSSLContextBuilder trustStorePasswordAppName(final String trustStorePasswordAppName) {
+            this.trustStorePasswordAppName = trustStorePasswordAppName;
+            return this;
+        }
+        
+        public ClientSSLContextBuilder privateKeyStore(final PrivateKeyStore privateKeyStore) {
+            this.privateKeyStore = privateKeyStore;
+            return this;
+        }
+        
+        public ClientSSLContextBuilder certAlias(final String certAlias) {
+            this.certAlias = certAlias;
+            return this;
+        }
+        
+        public SSLContext build() {
+            SSLContext context = null;
+            KeyStore keyStore = null;
+            KeyStore trustStore = null;
+            KeyManagerFactory kmf = null;
+            TrustManagerFactory tmf = null;
+            KeyManager[] keyManagers = null;
+            TrustManager[] trustManagers = null;
+            
+            if (keyStorePath == null && trustStorePath == null) {
+                return null;
+            }
+            
+            try {
+                if (keyStorePath != null) {
+                    LOGGER.info("createSSLContextObject: using SSL KeyStore path: " + keyStorePath);
```

Comment:
We should use {} instead of + so that we don't create the string if the info level is not enabled.